cardinal/system/system_game_setup.go

```diff
@@ -281,18 +319,29 @@ func (s *GameSetup) createPlace(roomID uint32, roomType enums.RoomType, dObjs [3
 	// Log successful creation
 	log.Printf("Room with ID %d created successfully, entity ID: %d", roomID, entityID)
 
-	// Log objects and directional objects added to the room
-	objDescriptions := make([]string, 0, len(objs))
+	/* LOGS FOR TESTING THAT THE ROOMS HAS THEIR OBJECTS, ETC. NEED TO BE POLISHED!
```

Comment:
we shouldn't really use logs for code testing we should test the actual created objects fro correctness (if that is what is being done here)

Reply:
I tested them through the test file. This was just for the terminal logs to see myself

Reply:
ok thats fine just checking but look at using slog or seomthing else like I said in another comment then you can set the severity of the logging

Reply:
Sure

Reply:
I implemented logrus logger so that I can have multiple levels of logs. I tested changing from InfoF, DebugF, WarnF but when running on dev they appeared anyways. Therefore I also created a method that returns a true or false value to show the all the logs if the the project is running on development or not.  The only tedious is that for the logs I had to indicate the method but this way I can control all the logs. The logs also have different colors depending the situation.

Reply:
yes they will on dev, that is expected behaviour, we dont want everything in production

Reply:
delete comments about LOGS...